src/utils/format_stackdriver/format_stackdriver.c

```diff
@@ -141,31 +141,49 @@ static int format_typed_value(yajl_gen gen, metric_t const *m,
     }
     break;
   }
-  case METRIC_TYPE_FPCOUNTER: {
+  case METRIC_TYPE_COUNTER: {
     /* Counter resets are handled in format_time_series(). */
-    assert(m->value.fpcounter >= start_value.fpcounter);
+    assert(m->value.counter >= start_value.counter);
 
-    fpcounter_t diff = m->value.fpcounter - start_value.fpcounter;
-    int status = json_string(gen, "doubleValue") || yajl_gen_double(gen, diff);
+    counter_t diff = m->value.counter - start_value.counter;
+    char integer[64] = {0};
+    ssnprintf(integer, sizeof(integer), "%" PRIu64, diff);
+
+    int status = json_string(gen, "int64Value") || json_string(gen, integer);
```

Comment:
As discussed in earlier PR, `ERR_COMBINE()` would be more appropriate as `||` reduces `int` return value to a `bool`.  That can be a nother PR though.

Reply:
Yep, not in this PR.